### DISCUSSION TITLE
About the next release

### DISCUSSION THREAD

**Discussion by lmagniez-sd** [Apparence-io/CamerAwesome#434] [General] [answered]
Are you planning to create a new release anytime soon? 
I've been wanting to update camerawesome to v2 but could not because of this issue: 
https://github.com/Apparence-io/CamerAwesome/issues/416
Thank you!

**Comment by g-apparence** [comment, marked as answer]
Hi, yes sorry for this late. 
Wanted to make more test before pushing this new version and had a lot of work on other projects. 
Will try to do it ASAP.
